examples/single_agent.py

```diff
@@ -1,68 +1,57 @@
 
 import gym
+import numpy as np
 
 from smarts.core.agent import Agent, AgentSpec
 from smarts.core.agent_interface import AgentInterface, AgentType
-from smarts.core.sensors import Observation
 from smarts.core.utils.episodes import episodes
 from smarts.env.wrappers.single_agent import SingleAgent
+from smarts.env.wrappers.standard_obs import StandardObs, StdObs
 
-# The following ugliness was made necessary because the `aiohttp` #
-# dependency has an "examples" module too.  (See PR #1120.)
 if __name__ == "__main__":
     from argument_parser import default_argument_parser
 else:
     from .argument_parser import default_argument_parser
 
 logging.basicConfig(level=logging.INFO)
 
-AGENT_ID = "SingleAgent"
 
-
-class ChaseViaPointsAgent(Agent):
-    def act(self, obs: Observation):
-        if (
-            len(obs.via_data.near_via_points) < 1
-            or obs.ego_vehicle_state.road_id != obs.via_data.near_via_points[0].road_id
-        ):
-            return (obs.waypoint_paths[0][0].speed_limit, 0)
-
-        nearest = obs.via_data.near_via_points[0]
-        if nearest.lane_index == obs.ego_vehicle_state.lane_index:
-            return (nearest.required_speed, 0)
+class ChaseWaypointsAgent(Agent):
+    def act(self, obs: StdObs):
+        cur_lane_index = obs.ego["lane_index"]
+        next_lane_index = obs.waypoints["lane_index"][0, 0]
 
         return (
-            nearest.required_speed,
-            1 if nearest.lane_index > obs.ego_vehicle_state.lane_index else -1,
+            obs.waypoints["speed_limit"][0, 0] / 2,
+            np.sign(next_lane_index - cur_lane_index),
         )
 
 
-def main(scenarios, sim_name, headless, num_episodes, seed, max_episode_steps=None):
+def main(scenarios, headless, num_episodes, max_episode_steps=None):
     agent_spec = AgentSpec(
         interface=AgentInterface.from_type(
             AgentType.LanerWithSpeed, max_episode_steps=max_episode_steps
         ),
-        agent_builder=ChaseViaPointsAgent,
+        agent_builder=ChaseWaypointsAgent,
     )
 
     env = gym.make(
         "smarts.env:hiway-v0",
         scenarios=scenarios,
-        agent_specs={AGENT_ID: agent_spec},
-        sim_name=sim_name,
+        agent_specs={"SingleAgent": agent_spec},
         headless=headless,
-        visdom=False,
-        fixed_timestep_sec=0.1,
         sumo_headless=True,
-        seed=seed,
-        # zoo_addrs=[("10.193.241.236", 7432)], # Sample server address (ip, port), to distribute social agents in remote server.
-        # envision_record_data_replay_path="./data_replay",
     )
 
-    # Wrap a single-agent env with SingleAgent wrapper to make `step` and `reset`
-    # output compliant with gym spaces.
-    env = SingleAgent(env)
+    # Convert SMARTS observations to standardized gym-compliant observations.
+    env = StandardObs(env=env)
+
+    # Convert `env.step()` and `env.reset()` from multi-agent interface to
+    # single-agent interface.
+    env = SingleAgent(env=env)
```

Comment:
Upcoming standard environments, for example `intersection_env` which is under development, will be wrapped with `FormatObs` inside the gym registration itself. 

I simply intended to show the usage of `StandardObs` here, but it might be unnecessary, hence it is removed from `single_agent.py` example.

Reply:
OK, that is reasonable.